ex36.py

```diff
@@ -174,6 +174,17 @@ def lion(current_count):
 	print """
 You walk into a brightly lit room,
 There is a huge great lion in this room.
+
+        @|\@@
+       -  @@@@
+      /7   @@@@
+     /    @@@@@@
+     \-' @@@@@@@@`-_______________
+      -@@@@@@@@@             /     
+ _______/    /_       ______/      |__________-
+/,__________/  `-.___/,_____________----------_)
+
+
```

Comment:
You could define your string literals as constants, and put them outside of the functions.

e.g.:
```